### ISSUE TITLE
0.6.1 : Chat command may crash server

### ISSUE THREAD

**Issue by laxa** [DDRace/teeworlds#76]
Problem with chat command, server may crash (some does, some other don't, I don't really know why).

**Comment by GreYFoX** [comment]
Can you please report at least 1 scenario that would lead to a crash?
EX: 1- do ....
2- do....
3- do
crash

**Comment by laxa** [comment]
I launch my homeserver, I go on it, I do whatever /cmd and server crash.

**Comment by GreYFoX** [comment]
Laxa i tried /rank /info /top5
only top5 didn't work and no crash
also /pause and /team # works

**Comment by laxa** [comment]
It crashes only after sv_map or reload apparently.

**Comment by Tsin** [comment]
/pause and /team crash the server

we done it yesterday many times with Hitomi and Save.

but we dont know if th couse was a "huge" map
or the map 0.6 (yes we started a 0.6 map on 0.6.1 ddrace 09a server) 

and untill anyone of us tried pause or joining team on that map (naremtest map = name of a map) the server was stable

but please keep in mind on dm1 everything works fine only on those bigger maps it crashes 

but i dont think the issue is because map was 0.6 

hitomo created HellGate 1.5 whole map was in 09a and also server crashed from /pause, /team x

also other important thing on my server with  that "naremtest" map we could use cheats like "left down right up"

but on hitomis server with hellgate 1.5 (0.6.1 09a map) any use of any cheat team pause = server crash

**Comment by Tsin** [comment]
Fox write if you want this issues to be here or should i make a special issue for every problem

**Comment by GreYFoX** [comment]
this is fixed, if there is stil lan issue open another ticket
